content/BCBestPractices/Readability/line-start-keywords/index.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Line Start Keywords"
+weight = 740
++++
+
+<_Created by Microsoft, Described by waldo_\>
+
+## Description
+The `end`, `if`, `repeat`, `for`, `while`, `else` and `case` statement should always start a line. 
+
+## Bad code
+
+```al
+    if IsContactName then ValidateContactName
+    else if IsSalespersonCode then ValidateSalespersonCode
+    else if IsSalesCycleCode then ValidatSalesCycleCode;
+```
+
+## Good code
+
+```al
+    if IsContactName then
+        ValidateContactName
+    else
+        if IsSalespersonCode then
+            ValidateSalespersonCode
+        else
+            if IsSalesCycleCode then 
+                ValidatSalesCycleCode;
+```
```

Comment:
Shouldn't You have parentheses for the FunctionNames?

```suggestion
```al
    if IsContactName then
        ValidateContactName()
    else
        if IsSalespersonCode then
            ValidateSalespersonCode()
        else
            if IsSalesCycleCode then 
                ValidatSalesCycleCode();
```
```



I know this is an example - but in this case (point taken) I would do

```AL
case true of
    IsContactName : ValidateContactName();
    IsSalesPersonCode : ValidateSalespersonCode()
    IsSalesCycleCode : ValidatSalesCycleCode();
end;
```

Reply:
That would be my assumption - My Suggestion is to add parentheses to the called function names

Reply:
oh .. my fault..